polkadot/primitives/src/v8/mod.rs

```diff
@@ -2239,7 +2239,7 @@ pub mod tests {
 
 	#[test]
 	fn test_backed_candidate_injected_core_index() {
-		let initial_validator_indices = bitvec![u8, bitvec::order::Lsb0; 0, 1, 0, 1];
+		let initial_validator_indices = bits![u8, Lsb0; 0, 1, 0, 1].to_bitvec();
```

Comment:
This change seems completely unrelated; why did you change this?

Reply:
Zed gave me compilation errors from these.

Reply:
And it is documented also in the prdoc entry.

Reply:
I.e. I know they are related but they still give compilation errors.

Reply:
It is documented in the prdoc, which I guess ultimately maps what a PR does and these compilation errors show red in Zed so thus keeping this side-trip.

Reply:
https://github.com/paritytech/polkadot-sdk/pull/6533/commits/e001c3971ef56c217030dd3b1e0060b4e758418d